lib/CL/pocl_cl.h

```diff
@@ -1568,6 +1570,16 @@ struct _cl_program {
   cl_build_status build_status;
   /* Use to store binary type */
   cl_program_binary_type binary_type;
+  /* total size of program-scope variables. This depends on alignments
```

Comment:
these must be available in `cl_program` for each device, because it's needed by `clGetProgramBuildInfo(program, device, ..., CL_​PROGRAM_​BUILD_​GLOBAL_​VARIABLE_​TOTAL_​SIZE)`